lib/widgets/app_header.dart

```diff
@@ -44,4 +68,39 @@ class AppHeader extends StatelessWidget implements PreferredSizeWidget {
       ),
     );
   }
-}
\ No newline at end of file
+
+  Widget getAuthButton(bool showAuthButton) {
```

Comment:
Wolałbym żeby widgety były klasami a nie funkcjami.

Reply:
to jest tylko wyekstrahowanie ifa do funkcji